src/main/java/mission1/step4/view/RenderView.java

```diff
@@ -0,0 +1,37 @@
+package mission1.step4.view;
+
+import mission1.step4.domain.car.Car;
+import mission1.step4.domain.car.CarEntry;
+
+import java.util.List;
+
+public class RenderView {
+
+    private static final String EXPRESSION = "-";
+
+    public static void showCarProgress(CarEntry carEntry) {
+        StringBuilder sb = new StringBuilder();
+
+        for (Car car : carEntry) {
+            sb.append(car.getName() + ": ");
+            for (int i = 0; i < car.getProgress(); i++) {
+                sb.append(EXPRESSION);
+            }
+            sb.append("\n");
+        }
+
+        System.out.println(sb.toString());
+    }
+
+    public static void showRaceWinner(List<Car> RaceWinnerList) {
```

Comment:
```suggestion
    public static void showRaceWinner(List<Car> raceWinnerList) {
```
자바에서는 변수 선언을 lowerCamelCase로 선언합니다 !